custom/enikshay/management/commands/run_result_summary_display_migration.py

```diff
@@ -0,0 +1,37 @@
+from custom.enikshay.management.commands.utils import BaseEnikshayCaseMigration
+
+DATAMIGRATION_CASE_PROPERTY = 'datamigation_result_summary_display'
+RESULT_SUMMARY_DISPLAY = 'result_summary_display'
+
+
+class Command(BaseEnikshayCaseMigration):
+    case_type = 'test'
+    case_properties_to_update = [
+        RESULT_SUMMARY_DISPLAY,
+    ]
+    datamigration_case_property = DATAMIGRATION_CASE_PROPERTY
+    include_public_cases = True
+    include_private_cases = False
+
+    @staticmethod
+    def get_case_property_updates(test, domain):
+        if (
+            test.get_case_property(DATAMIGRATION_CASE_PROPERTY) == 'yes'
+            or not (
+                test.get_case_property('result_recorded')
+                and not test.get_case_property(RESULT_SUMMARY_DISPLAY)
+                and not test.get_case_property('test_type_value')
+            )
+        ):
+            return {}
+
+        return {
+            RESULT_SUMMARY_DISPLAY: '\n'.join(filter(None, [
+                'TB Detected' if test.get_case_property('result') == 'tb_detected' else 'TB Not Detected',
```

Comment:
@nickpell 
I think we want to actually have it be:
if test.get_case_property('result') == 'tb_detected':
    value = 'TB Detected'
elif test.get_case_property('result') == 'tb_not_detected':
    value = 'TB Not Detected'
else:
    value = None